azure-kusto-ingest/example.js

```diff
@@ -132,34 +58,80 @@ const streamingIngestClient = new StreamingIngestClient(
     props2
 );
 
-// Ingest from file with either file path or FileDescriptor
-streamingIngestClient.ingestFromFile("file.json", null, (err) => {
-    if (err) {
+startStreamingIngestion();
+
+async function startIngestion() {
+    console.log("Ingest from file");
+    try {
+        await ingestClient.ingestFromFile("file.json");
+        console.log("Ingestion done?");
+
+        await waitForStatus();
+    }
+    catch (err) {
         console.log(err);
     }
 
-    console.log("Ingestion done");
-});
-
-// Ingest from stream with either ReadStream or StreamDescriptor
-const stream = fs.createReadStream("file.json");
+    try {
+        await ingestClient.ingestFromBlob(new BlobDescriptor("https://<account>.blob.core.windows.net/<container>/file.json.gz", 1024 * 50 /* 50MB file */));
```

Comment:
Consider adding consts (or even into config these values)

Reply:
this is an example, I think it's ok to demonstrate we want the size in bytes